utils/build.gradle.kts

```diff
@@ -0,0 +1,19 @@
+@Suppress("DSL_SCOPE_VIOLATION") // TODO: Remove once KTIJ-19369 is fixed
+plugins {
+    id("java-library")
+    alias(libs.plugins.kotlin.jvm)
+}
+
+java {
```

Comment:
remove it once you migrate to `multiplatform-library-convention`

Reply:
done